cmake/GitVersion.cmake

```diff
@@ -46,25 +46,21 @@ function (get_version_tag_from_git GIT)
         message(STATUS "You are currently on commit ${COMMIT}")
 
         # Get all the tags
-        execute_process(COMMAND "${GIT}" rev-list --tags --max-count=1 --abbrev-commit RESULT_VARIABLE RET OUTPUT_VARIABLE TAGGEDCOMMIT OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
This patch is also different from the original due to Monero PR 6036 having not been merged yet. Can you cherry-pick it first?

Reply:
Revised to include 6036.